### PR TITLE
Updates to SB - 2.0.x branch

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -186,12 +176,12 @@
     <jandex.version>2.0.0.Final</jandex.version> <!-- For Hibernate core -->
 
     <!-- Spring Ecosystem -->
-    <spring-boot.version>2.0.0.RC1</spring-boot.version>
-    <spring-cloud-config.version>2.0.0.M5</spring-cloud-config.version>
-    <spring-cloud-netflix.version>2.0.0.M5</spring-cloud-netflix.version>
-    <spring-cloud-sleuth.version>2.0.0.M5</spring-cloud-sleuth.version>
+    <spring-boot.version>2.0.1.RELEASE</spring-boot.version>
```
Comment (metacosm) [inline pom.xml]: 2.0.2 has been released with CVE fixes so we should probably use that version.

**Comment by geoand** [comment]
@metacosm PR updated

**Comment by metacosm** [comment]
By the way, why the update now?

**Comment by geoand** [comment]
Well, we need to prepare the ground for an SB 2.x release don't we?

**Comment by metacosm** [comment]
True, but was just wondering why now… :)

**Comment by metacosm** [comment]
Also, considering that other dependencies need to be cleaned-up / removed, cf. `openshift-client` for example. So I was wondering about the timing and scope of this PR.

**Comment by geoand** [comment]
Just came across it and wanted to fix the low hanging fruit

**Comment by metacosm** [comment]
OK. Was wondering if someone needed these changes.

**Comment by geoand** [comment]
Not that I remember

**Pull request merged**